index.html

```diff
@@ -703,15 +702,25 @@
           context</a>.
         </p>
         <p>
-          The <dfn>set of presentations</dfn>, initially empty, contains the
-          <a>presentations</a> created by the <a>controlling browsing
-          contexts</a> for the controlling user agent (or a specific user
-          profile within that user agent). The <a>set of presentations</a> is
-          represented by a list of tuples, where each tuple consists of a
-          <a>presentation URL</a>, a <a>presentation identifier</a>, and the
-          <a>presentation connection</a> itself. The <a>presentation URL</a>
-          and <a>presentation identifier</a> uniquely identify the
-          <a>presentation</a>.
+          The <dfn>set of controlling presentations</dfn>, initially empty,
```

Comment:
Perhaps the "set of connected presentations" or "set of controlled presentations"? The presentation is not necessarily "controlling" a browsing context connected to it; usually I think of it the other way around.